src/cyw43_spi.h

```diff
@@ -34,6 +34,11 @@
 #ifndef CYW43_INCLUDED_CYW43_SPI_H
 #define CYW43_INCLUDED_CYW43_SPI_H
 
+#include "cyw43_internal.h"
```

Comment:
That seems strange that a public header needs to include an internal header... need to think about this some more.

Reply:
> That seems strange that a public header needs to include an internal header... need to think about this some more.

Agreed. The supposedly internal cyw43_int_t in the supposedly public API makes for a visibilty mismatch.

I'll handle the other remarks in the meantime.